scss/main.scss

```diff
@@ -73,6 +72,61 @@ $grey: #C4C4C4;
   animation-delay: -50s;
 }
 
+#cloud2 {
```

Comment:
hmm theres some weird behavior where white space/horizontal scroll is generated when the clouds move off screen. i was able to fix it by adding 
".hero {
  max-width: 100vw;
  overflow: hidden;
  position: relative;
}"
but because of the position: relative, you'll have to adjust the margin-top for each cloud :((( sorry :(